steps/addressSteps.ts

```diff
@@ -0,0 +1,14 @@
+import DeleteAddressPage from '../pages/deleteAddressPage';
+import HomePage from '../pages/homePage';
+
+export class AddressSteps {
+  constructor(
+    private homePage: HomePage,
+    private deleteAddressPage: DeleteAddressPage
+  ) {}
+
+  async deleteAddressByStreet(streetAddress: string) {
```

Comment:
const homePage = new HomePage(page);
await this.homePage.clickDeleteAddressBtnByStreet(streetAddress);